src/meta/src/controller/catalog.rs

```diff
@@ -774,7 +774,7 @@ impl CatalogController {
         .await?;
         pb_connection.id = conn_obj.oid as _;
         let connection: connection::ActiveModel = pb_connection.clone().into();
-        connection.insert(&txn).await?;
+        Connection::insert(connection).exec(&txn).await?;
```

Comment:
Is this just compatibility changes from the seaorm version upgrade?

Reply:
Nope, the former one will return the inserted model, which is not necessary and brings some overhead. For example for MySQL, it generates another full query after inserting.